internal/log/grpc.go

```diff
@@ -0,0 +1,84 @@
+package log
+
+import (
+	"fmt"
+
+	"github.com/rs/zerolog"
+	"google.golang.org/grpc/grpclog"
+)
+
+func init() {
+	grpclog.SetLoggerV2(&grpcLogger{})
+}
+
+type grpcLogger struct{}
+
+func (c *grpcLogger) Info(args ...interface{}) {
+	if GetLevel() == zerolog.DebugLevel {
```

Comment:
does this need to be something like `<=`? I think zerolog has `trace` too

Reply:
Debug is zero in zero log but sure to be future proof we can do <=

Reply:
// TraceLevel defines trace log level.
	TraceLevel Level = -1

it seems like they added it indeed as -1